### ISSUE TITLE
False positive on ternary operator

### ISSUE THREAD

**Issue by mockdeep** [sds/haml-lint#46]
I have this code:

``` haml
%em.text-muted
  - if group.sequential?
    Routed to Multiple Recipients
  - elsif group.all_at_once?
    Alternate Emails Uploaded
```

And I'm getting this message from `haml-lint`:

```
[W] RuboCop: Favor the ternary operator (?:) over if/then/else/end constructs.
```

The same code represented in pure ruby passes rubocop, so it seems like an issue in `haml-lint`.
